README.md

```diff
@@ -1,2 +1,24 @@
-# trinity-carpool
-A carpool management app by the students of Austin's Trinity Episcopal School.
+# Trinity Carpool
+### A carpool management app designed and created by the students of Austin's Trinity Episcopal School. 
+-------------------------------------------------------------------------------
+***We are trying to...***
+<br>Create an app that helps carpool run faster, smoother, and more efficiently. 
+This is so that we get longer recess.
+-------------------------------------
+This checkout system is designed to get kids out of the carpool room 
+*as quickly and effeciently as possible.* For example, if two kids are carpooling 
+together, and are not related, the parent picking them up would say the names of 
+the kids being picked up. Then, the kids will get a notification, and go outside. 
+After the parent has the kids, the app will let you check out. Once you have checked out, 
+the teachers will know you have gone. <br>
```

Comment:
The teachers will know you have left

Reply:
I like this, this is a good idea! Also should we capitalize the first words?